lib/rubocop/cop/rails/enum_keyword_args.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Looks for enums written with keyword arguments.
+      #
+      # Defining enums with keyword arguments is deprecated
+      # and will be removed in Rails 7.3.
+      #
+      # Positional arguments should be used instead:
+      #
+      # @example
+      #   # bad
+      #   enum status: { active: 0, archived: 1 }, _prefix: true
+      #
+      #   # good
+      #   enum :status, { active: 0, archived: 1 }, prefix: true
+      #
+      class EnumKeywordArgs < Base
+        extend AutoCorrector
```

Comment:
I think that it has been supported since Rails 5.0:

```suggestion
        extend AutoCorrector
        extend TargetRailsVersion

        minimum_target_rails_version 5.0
```